examples/test/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+cc_test(
+    name = "simple_test",
+    srcs = ["simple_test.cpp"], 
+    deps = [
+        "@catch2//:catch2",        
+        "@catch2//:catch2_main",   
+    ],
+    includes = ["@catch2//:src"],  
```

Comment:
```starlark
includes = ["@catch2//:src"],
``` 
I have never seen this being used. I think this is used to bring outside libraries to compile. This file compiles without this line. Remove this.

Reply:
The compiler initially had trouble finding the header and I added this thinking the include folder was missing, the problem got resolved but I forgot to remove this line. I have removed this now.